### PR TITLE
build: Bump Kubernetes version to 1.16.15

### PR DESCRIPTION
**Pull request by TeddyAndrieux**
Bump Kubernetes version to 1.16.15

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,8 +6,8 @@
 - [#2572](https://github.com/scality/metalk8s/issues/2572) - Bump CoreDNS
 version to 1.6.2 (PR [#2575](https://github.com/scality/metalk8s/pull/2575))
 
-- [#2674](https://github.com/scality/metalk8s/issues/2674) - Bump K8S version
```
Comment (gdemonet) [inline CHANGELOG.md:9]: Let's keep the issue reference, since it mentions some fixed CVEs.

Comment (TeddyAndrieux) [reply CHANGELOG.md:9]: :ok:

**Review by gdemonet — APPROVED** [review]

**Pull request merged**